src/components/plotCard.vue

```diff
@@ -0,0 +1,46 @@
+<template lang="pug">
+q-card(bordered flat)
+  .q-pa-sm
+    .row.items-center
+      q-icon.q-mr-sm(color="grey" name="downloading" size="40px")
+      h6.text-weight-light {{ lang.plot }}
+    q-separator.q-mt-xs
+    .row.items-center.q-mt-sm
+      .col-auto.q-mr-md(v-if="plot.state == 'finished'")
+        q-icon(color="green" name="done" size="40px")
+      .col-auto.q-mr-md(v-if="plot.state == 'verifying'")
+        q-spinner-box(color="grey" size="40px")
+      .col-auto.q-mr-md(v-if="plot.state == 'starting'")
+        q-spinner-orbit(color="grey" size="40px")
+      .col
+        .text-weight-light {{ lang.status }}
+        p {{ plot.message }}
+    .row.items-center.q-mt-sm
+      .col-auto.q-mr-md
+        q-icon(color="black" name="storage" size="40px")
+      .col
+        .text-weight-light {{ lang.allocated }}
+        p {{ util.toFixed(config?.plot?.sizeGB, 2) }} GB
+</template>
+
+<script lang="ts" >
+import { defineComponent } from "vue"
+import * as util from "src/lib/util"
+import * as global from "src/lib/global"
+const lang = global.data.loc.text.dashboard
+
+export default defineComponent({
+  data() {
+    return { lang, util }
+  },
+  props: {
+    plot: { type: Object, required: true },
+    config: { type: Object, required: true },
+  },
+})
+</script>
+
+
+
+<style>
+</style>
```

Comment:
Please configure your IDE to always add new line at the end of the file.

Reply:
done